app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -72,6 +72,39 @@
             errorMsg="Please enter a description"
           />
 
+          <!-- Grant website -->
+          <BaseInput
+            v-model="form.website"
+            description="Your grant's website"
+            id="grant-website"
+            label="Grant website"
+            :rules="isValidUrl"
+            errorMsg="Please enter a valid URL"
+            :required="false"
+          />
+
+          <!-- Grant github -->
+          <BaseInput
+            v-model="form.github"
+            description="Your grant's github"
+            id="grant-github"
+            label="Grant github"
+            :rules="isValidUrl"
+            errorMsg="Please enter a valid URL"
+            :required="false"
+          />
+
+          <!-- Grant twitter handle -->
+          <BaseInput
+            v-model="form.handle"
+            description="Your grant's twitter handle"
+            id="grant-handle"
+            label="Grant twitter"
+            :rules="isValidUrl"
```

Comment:
While we settled on *storing* a twitter URL in the metadata, I don't think the form should *take* a URL from the user. Instead, I think we should ask them to input their handle— which we would accept with or without the "@" sign— and then normalize it and append it to `https://twitter.com/` to form the expected URL.

Reply:
should this validation be done inside GrantRegistryGrantDetail.vue / GrantRegistryNewGrant.vue or in another file (like utils)?

Reply:
Probably utils since you'll need to do the normalization both on grant creation and when editing